docs/reference/ccr/getting-started.asciidoc

```diff
@@ -304,37 +306,34 @@ POST /server-metrics-follower/_ccr/unfollow
 ====
 
 [[ccr-getting-started-auto-follow]]
-==== Automatically create follower indices
+==== Create an auto-follow pattern to replicate time series indices
 Create <<ccr-auto-follow,auto-follow patterns>> to automatically follow time
-series indices that are periodically created in a remote cluster (such as daily
-{beats} indices).
+series indices that are created in a remote cluster. Whenever the name of a new index on the remote cluster matches the auto-follow pattern, a corresponding follower index is added to the local cluster.
```

Comment:
I wonder if it's worth saying something about the auto-follow pattern essentially matching the index pattern specified in the template used to create the time series indices on the remote?